relevanceai/operations/cluster/ops.py

```diff
@@ -80,40 +64,30 @@ class ClusterOps(PartialClusterOps, SubClusterOps):
 
     def __init__(
         self,
-        alias: str,
         project: str,
         api_key: str,
         firebase_uid: str,
-        model: Union[BatchClusterBase, ClusterBase, CentroidClusterBase] = None,
-        dataset_id: Optional[str] = None,
-        vector_fields: Optional[List[str]] = None,
+        model: Union[BatchClusterBase, ClusterBase, CentroidClusterBase, Any] = None,
```

Comment:
This needs to contain `vector_fields` in order to allow for reload. intended behavior is: 

```
cluster_ops = client.ClusterOps(alias="kmeans-3", vector_fields="sample")
cluster_ops.list_closest_to_center()
# prints me a link I can click
```

```
from sklearn.cluster import KMeans
kmeans_model = KMeans()

kmeans_model = KMeans(n_clusters=3)
cluster_ops = client.ClusterOps(model=kmeans_model)
# List closest to center
cluster_ops.fit_predict(dataset_id="sample_dataset", vector_fields=['sample_vector_'])
# potentially generate alias
# add cluster metadata to metadata
```

Reply:
okay but I dont like it